Users/Install.rst

```diff
@@ -184,8 +191,13 @@ options are documented in the
     <http://stackoverflow.com/questions/14149422/using-pip-behind-a-proxy>`_
     for further clarification.
 
-Dependencies
-------------
+Optional Dependencies
+---------------------
+
+Those dependencies are not mandatory. You may do this only if you want
+to install all the bears. The bear will also ask for the missing packages
+in case something is missing so in case you want to install one or two bears
+it is not necessary to do this step.
```

Comment:
maybe add another paragraph stating that the Gemfile etc. is in the coala-bears repository, they don't need to install from source to use it but they need a clone to do those commands in